components/_provisional.scss

```diff
@@ -286,13 +286,48 @@
 		}
 	}
 
-	/* Theme/topic decorative image */
-	&.gc-topic-bg div {
-		background: no-repeat;
-		margin-left: -15px;
-		min-height: 200px;
-		overflow: hidden;
-		width: calc(50vw - 15px);
+	&.gc-most-requested {
+		border: none;
+
+		h2 {
+			font-size: $font-size-h4;
+			margin-top: $space-md;
+		}
+		ul {
+			margin-bottom: $space-md;
+			margin-top: $space-md;
+
+			li {
+				font-family: "Lato", sans-serif;
+				font-size: $small-size;
+				font-weight: $bold-weight;
+				line-height: 1.5em;
+			}
+		}
+	}
+
+	&.gc-featured-bnr {
```

Comment:
@duboisp What about `.gc-featbnr`?

Reply:
@GormFrank and @duboisp the instance from the institutional landing page is called "Featured link" and since there will only be a link without images, I suggest renaming the component as follow: gc-featuredLink .